docs/distributions/fedora/installation.md

```diff
@@ -2,7 +2,7 @@
 
 Many thanks to Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora).
 
-If you need a more updated kernel, use the iso from [here](https://github.com/sharpenedblade/t2linux-fedora-iso/releases). Follow the instalation instructions below until step 5, then install like a normal Fedora iso. If you do not know how to install Fedora normally, read [this](https://docs.fedoraproject.org/en-US/fedora/latest/install-guide/). Remember to follow the [Wi-Fi guide](https://wiki.t2linux.org/guides/wifi-bluetooth/).
```

Comment:
Follow the installation instructions......

Why did you remove this line?

Also, what exactly have you made clear in this PR?

Reply:
>Why did you remove this line?

Becuase with the updated instruction you have to follow everything for my ISO.
>Also, what exactly have you made clear in this PR?

Like 2 people dmed me about mikeeqs ISO, and thought it was mine, so I made the link say `ISO from sharpenedblade`.